File: js/stockPicker.js
exports.picker = function(prices) {
    let profitChances = []
    let dayCombos = []
    let value = [prices[1] - prices[0]]
    let pricesTwo;
    let profit = 0;
    let answer = [prices[1],prices[0]]
    pricesTwo = prices.slice(0)
    
    for (let i = 0; i<prices.length; i++) {
      pricesTwo.shift()
      for (let j = 0; j<pricesTwo.length; j++) {
        profit = pricesTwo[j] - prices[i];
        if (profit > value) {
          value = profit;
          answer = [prices[i],pricesTwo[j]]
        }
      } 
      profitChances.push(value);
      dayCombos.push(answer);
    }
    let biggestReturn = Math.max(...profitChances)
    let indexHolder = profitChances.indexOf(biggestReturn)
    let buySell = dayCombos[(indexHolder)]
    let buy = prices.indexOf(buySell[0])
    let sell = prices.indexOf(buySell[1])
    return [buy,sell]
  }

File: python/stock_picker.py
def picker(prices):
    profit_chances = []
    day_combos = []
    prices_two = prices.copy()
    answer = [prices[1] - prices[0]]
    for x in prices:
        profit = prices[1] - prices[0]
        prices_two.pop(0)
        for y in prices_two:
            value = int(y) - int(x)
            if int(value) > int(profit):
                profit = value
                answer = [x, y]
        profit_chances.append(profit)
        day_combos.append(answer)
    biggest_return = max(profit_chances)
    index_holder = profit_chances.index(biggest_return)
    buy_sell = day_combos[int(index_holder)]
    buy = prices.index(int(buy_sell[0]))
    sell = prices.index(int(buy_sell[1]))
    return [buy,sell]
